src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -522,10 +522,10 @@ private void validateBeforeApproval(Organization organization, Invoice invoice,
     if (organization == null) {
       throw new HttpException(404, ORG_NOT_FOUND);
     }
-    if (!organization.getIsVendor()) {
+    if (Boolean.FALSE.equals(organization.getIsVendor())) {
       var param = new Parameter().withKey("organizationId").withValue(organization.getId());
       var error = ORG_IS_NOT_VENDOR.toError().withParameters(List.of(param));
-      logger.error(JsonObject.mapFrom(error).encodePrettily());
+      logger.error("validateBeforeApproval:: Organization is not vendor. '{}'", param);
```

Comment:
the same here, just adding organizationId would be more simpler and readable

Reply:
Thanks, improved log.errors